tests/integration/runk/runk-tests.bats

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023,2024 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will validate runk with containerd
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/../../metrics/lib/common.bash"
+
+setup_file() {
+	export RUNK_BIN_PATH="/usr/local/bin/runk"
+	export TEST_IMAGE="docker.io/library/busybox:latest"
```

Comment:
Is quay.io/prometheus/busybox preferrable, to work around image repository limits?